test/js/web/timers/setTimeout-cpu-fixture.js

```diff
@@ -1 +1,3 @@
-setTimeout(() => {}, 3_000); // we use 3s so we can reproduce better the issue #7790
+setTimeout(() => {
```

Comment:
Why are these files labeled as generated on Github?

Reply:
Because the filename matches the pattern in https://github.com/oven-sh/bun/blob/912bccb6243bbc169c0d92e23dc1be7f01e44fba/.gitattributes#L26